### ISSUE TITLE
Swiftlint does not allow multiple configs from command line.

### ISSUE THREAD

**Issue by luispadron** [realm/SwiftLint#3652]
### New Issue Checklist

- [X] Updated SwiftLint to the latest version
- [X] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

According to the documentation:

> Command Line
> Instead of just providing one configuration file when running SwiftLint via the command line, you can also pass a hierarchy, where the first configuration is treated as a parent, while the last one is treated as the highest-priority child.
> 
> A simple example including just two configuration files looks like this:
> 
> swiftlint --config ".swiftlint.yml .swiftlint_child.yml"

However, tying something like: `swiftlint --config ".swiftlint.yml .swiftlint_child.yml"` fails with:

```
error: SwiftLint Configuration Error: Could not read file at path: /Users/luis.padron/development/clide/.swiftlint.yml .swiftlint_child.yml
Could not read configuration: file Configuration.swift, line 237
zsh: abort      swiftlint --config ".swiftlint.yml .swiftlint_child.yml"
```

This indicates to me that the `--config` option is bein interpreted as a single path instead of multiple paths. 
Which seems related to: #3471 

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
 Δ clide (luis/project-lint) > swiftlint --config ".swiftlint.yml .swiftlint_child.yml"
error: SwiftLint Configuration Error: Could not read file at path: /Users/luis.padron/development/clide/.swiftlint.yml .swiftlint_child.yml
Could not read configuration: file Configuration.swift, line 237
zsh: abort      swiftlint --config ".swiftlint.yml .swiftlint_child.yml"
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? - 0.43.1
* Installation method used (Homebrew, CocoaPods, building from source, etc)? - brew

**Comment by tinder-maxwellelliott** [comment]
I am experiencing the same issue

**Comment by dirtyhabits97** [comment]
This should do the trick according to the [swift-argument-parser docs](https://github.com/apple/swift-argument-parser/blob/main/Documentation/02%20Arguments%2C%20Options%2C%20and%20Flags.md#alternative-array-parsing-strategies):

```
swiftlint --config .swiftlint.yml --config .swiftlint_child.yml
```
